.travis.yml

```diff
@@ -23,10 +23,10 @@ cache:
     - $HOME/elfutils
 script:
   - curl -sSL https://github.com/maidsafe/QA/raw/master/Bash%20Scripts/Travis/build_and_run_tests.sh | bash
-#  - cd $TRAVIS_BUILD_DIR && cargo build --example ci_test
-#  - cd $TRAVIS_BUILD_DIR && cargo build --example key_value_store
-#  - cd $TRAVIS_BUILD_DIR && cargo run --release --example ci_test > output.log
-#  - cat output.log
+  - cd $TRAVIS_BUILD_DIR && cargo build --example ci_test
+  - cd $TRAVIS_BUILD_DIR && cargo build --example key_value_store
+  - cd $TRAVIS_BUILD_DIR && cargo run --release --example ci_test > output.log
+  - cat output.log
```

Comment:
Those last two lines had already been disabled before. Let's only re-enable the first two (building) and not run the ci_test on Travis for now.